nautobot/ipam/models.py

```diff
@@ -783,11 +806,8 @@ def siblings(self, include_self=False):
         return query
 
     def get_child_ips(self):
-        """
-        Return all IPAddresses within this Prefix and VRF. If this Prefix is a container in the global table, return
-        child IPAddresses belonging to any VRF.
-        """
-        return IPAddress.objects.net_host_contained(self.prefix)
+        """Return all IPAddresses within this Prefix and Namespace."""
```

Comment:
```suggestion
        """Return all IPAddresses directly contained within this Prefix and Namespace."""
```

With this change we now only return directly descended IPs now, rather than all IPs contained within this prefix?